CHANGELOG.mkd

```diff
@@ -5,8 +5,13 @@ Unreleased
 ----------
```

Comment:
oh yeah, and would you mind putting this back to just `----`?

Reply:
When rendered, this makes a line just like the shorter version. When unrendered, I kinda like that it's the same length as the word 🤷‍♀️ But I can change it if you want.

Reply:
oh interesting. then yeah, sure, we can keep it like this.